From: win7-support-bot <win7@cromite.org>
Date: Sun, 26 Jul 2026 00:00:00 +0000
Subject: WIN7: testing-variations / dwrite

Files:
 testing/variations/fieldtrial_testing_config.json

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
diff --git a/testing/variations/fieldtrial_testing_config.json b/testing/variations/fieldtrial_testing_config.json
index 26b26f5b6ed41..456bcd08163d5 100644
--- a/testing/variations/fieldtrial_testing_config.json
+++ b/testing/variations/fieldtrial_testing_config.json
@@ -9989,42 +9989,6 @@
             ]
         }
     ],
-    "FontDataServiceAllWebContents": [
-        {
-            "platforms": [
-                "windows"
-            ],
-            "experiments": [
-                {
-                    "name": "DWrite_Enabled",
-                    "params": {
-                        "typeface": "DWrite"
-                    },
-                    "enable_features": [
-                        "FontDataServiceAllWebContents"
-                    ]
-                },
-                {
-                    "name": "Freetype_Enabled",
-                    "params": {
-                        "typeface": "Freetype"
-                    },
-                    "enable_features": [
-                        "FontDataServiceAllWebContents"
-                    ]
-                },
-                {
-                    "name": "Fontations_Enabled",
-                    "params": {
-                        "typeface": "Fontations"
-                    },
-                    "enable_features": [
-                        "FontDataServiceAllWebContents"
-                    ]
-                }
-            ]
-        }
-    ],
     "FontDataServiceChromeOS": [
         {
             "platforms": [
